flutter_local_notifications_windows/lib/src/plugin/base.dart

```diff
@@ -0,0 +1,67 @@
+import "package:flutter_local_notifications_platform_interface/flutter_local_notifications_platform_interface.dart";
+import "package:timezone/timezone.dart";
+
+import "../details.dart";
+
+export "package:flutter_local_notifications_platform_interface/flutter_local_notifications_platform_interface.dart";
+export "package:timezone/timezone.dart";
+
+/// The Windows implementation of `package:flutter_local_notifications`.
+abstract class WindowsNotificationsBase extends FlutterLocalNotificationsPlatform {
+  /// Initializes the plugin. No other method should be called before this.
+  Future<bool> initialize(
+    WindowsInitializationSettings settings, {
+    DidReceiveNotificationResponseCallback? onNotificationReceived,
+  });
+
+  /// Releases any resources used by this plugin.
+  void dispose();
+
+  /// The raw XML passed to the Windows API.
```

Comment:
Something seems off here. The comments say it's for the raw XML as though this method  returns the XML representation of the notification. However, it looks like it's to show a notification based on the raw XML representation. Even if it's for showing a notification based on the raw XML representation, is there a reason why this method exists? Would imply it's not possible to do other things like schedule a notification using the raw XML representation. I would've thought that the raw XML is an alternative representation. If so, semantically speaking  it would IMO be more appropriate that the `WindowsNotificationDetails` provides a named constructor that allows providing a raw XML

Reply:
The original branch I worked with did have a `WindowsNotificationDetails.fromXml` constructor, however, I decided to remove it. The XML itself decides the title, body, and payload, and so I felt it would be off-putting to be able to say `show(title, body, payload, someOtherXmlThatOverridesEverything)`. 

That's why `notificationToXml` exists, because the XML is a combination of the title, body, payload, and all the other details -- even if none are provided. I figured it was fine to start off without the ability to schedule XML for now, but I can add that in too, it won't be much work.

Reply:
I added `zonedScheduleRawXml` to the API, which is the same as `zonedSchedule` but XML replaces title, body, and payload. Resolving but feel free to re-open